src/components/media-views.js

```diff
@@ -108,19 +108,19 @@ async function createGIFTexture(url) {
  * @param {string} src - Url to a video file.
  * @returns {Element} Video element.
  */
-async function createVideoEl() {
-  const videoEl = document.createElement("video");
-  videoEl.setAttribute("playsinline", "");
-  videoEl.setAttribute("webkit-playsinline", "");
+async function createVideoOrAudioEl(type) {
```

Comment:
I know this is not new.. but why is this function async? Doesn't seem to return a promise or do anything async.

Reply:
no idea, changing it.